RecoEgamma/EgammaElectronProducers/plugins/LowPtGsfElectronIDProducer.cc

```diff
@@ -212,16 +124,19 @@ void LowPtGsfElectronIDProducer::produce(edm::StreamID, edm::Event& event, const
     edm::ValueMap<float>::Filler filler(*ptr);
     filler.insert(gsfElectrons, output[iname].begin(), output[iname].end());
     filler.fill();
-    reco::GsfElectronRef ele(gsfElectrons, 0);
+    //reco::GsfElectronRef ele(gsfElectrons, 0);
     event.put(std::move(ptr), names_[iname]);
   }
 }
 
-double LowPtGsfElectronIDProducer::eval(const std::string& name, const reco::GsfElectronRef& ele, double rho) const {
+double LowPtGsfElectronIDProducer::eval(const std::string& name,
+                                        const edm::Ptr<reco::GsfElectron> ele,
```

Comment:
consider `const edm::Ptr<reco::GsfElectron>&` as good practice to avoid making a copy